ntoskrnl/inbv/inbv.c

```diff
@@ -1057,18 +1082,19 @@ DisplayBootBitmap(IN BOOLEAN TextMode)
 
         if (!RotBarThreadActive && TempRotBarSelection != RB_UNSPECIFIED)
         {
-            /* Start rotation bar animation thread */
-            PltRotBarStatus = 0;
+            /* Start the animation thread */
+            PltRotBarStatus = 1;
 
             Status = PsCreateSystemThread(&ThreadHandle,
-                                          SYNCHRONIZE,
+                                          THREAD_ALL_ACCESS,
```

Comment:
Ok here I've just modified because in other calls to that function we use all access, but I think actually that it should not matter for us which value we use.

Reply:
Allright then.

Reply:
I don't agree with that change. The rule is: minimum required permissions. People just use other values because they're lazy.

Reply:
Then we can use "0" :)

Reply:
That'll probably fail, but READ_CONTROL or SYNCHRONIZE will do fine.